BAZProjectC1/Components/BasicInfoMovie/movieImageView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  movieImageView.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 28/09/22.
+//
+
+import Foundation
+import UIKit
+
+open class movieImageView: UIView {
```

Comment:
Es necesario que esta clase sea `open`?

Recuerda que la convención para los nombres de clases es UpperCammelCase

```suggestion
class MovieImageView: UIView {
```